### PR TITLE
Add support for LBaaSv2 from contrail

### PR DESCRIPTION
**Pull request by mnederlof**

### CODE REVIEW

**Comment by pupapaik** [comment]
@pjediny we discuss this. There is not needed to put v2 or v1, because v1 was deprecated by Mitaka relase. We will bring proper fix this week.

File: README.rst [hunk 1]
```diff
@@ -89,6 +89,19 @@ pillar should be define only if it should be disabled.
...
+If you want to enable contrail LBaaSv2, then configure pillar like this:
+
+.. code-block:: yaml
+
+  neutron:
+    server:
+      lbaas:
+        enabled: true
+        providers:
+          Opencontrail:
```
Comment (epcim) [inline README.rst]: 'Opencontrail' should rather be lowercase 'opencontrail'

File: neutron/files/mitaka/neutron-server.conf.Debian [hunk 2]
```diff
@@ -28,14 +28,19 @@ bind_port = {{ server.bind.port }}
 
 {% if server.backend.engine == "contrail" %}
 
-# TEMPORARY - until neutron v2 contrail package would be supported
-#api_extensions_path = extensions:/usr/lib/python2.7/dist-packages/neutron_plugin_contrail/extensions:/usr/lib/python2.7/dist-packages/neutron_lbaas/extensions
+{% if server.get('lbaas', {}).get('providers', {}).get('Opencontrail', {}).get('engine', 'contrail') == 'contrail-lbaasv2' %}
```
Comment (epcim) [inline neutron/files/mitaka/neutron-server.conf.Debian]: 'Opencontrail' should rather be lowercase 'opencontrail'

File: neutron/files/mitaka/neutron-server.conf.Debian [hunk 3]
```diff
@@ -28,14 +28,19 @@ bind_port = {{ server.bind.port }}
...
+api_extensions_path = extensions:/usr/lib/python2.7/dist-packages/neutron_plugin_contrail/extensions:/usr/lib/python2.7/dist-packages/neutron_lbaas/extensions
+service_plugins = neutron_plugin_contrail.plugins.opencontrail.loadbalancer.v2.plugin.LoadBalancerPluginV2
```
Comment (epcim) [inline neutron/files/mitaka/neutron-server.conf.Debian]: pls. make shure, (post confirmation) the plugin paths works with juniper distribution packaging as well.

**Comment by epcim** [comment]
-1
Missing tests/pillar/ + update suite in .kitchen.yml

**Comment by mnederlof** [comment]
It is integrated in master :)

**Pull request closed without merging**